apps/explorer/tests/utils/test_utils.py

```diff
@@ -0,0 +1,160 @@
+import pandas
+import yaml
+
+from ...utils import export_pixelsets
+from apps.core.models import PixelSet
+from apps.core.tests import CoreFixturesTestCase
+from apps.core import factories
+
+
+class ExportPixelsSetTestCase(CoreFixturesTestCase):
+
+    def _assert_archive_is_valid(self, zip_archive):
+        assert len(zip_archive.namelist()) == 2
+        assert 'meta.yaml' in zip_archive.namelist()
+        assert 'pixels.csv' in zip_archive.namelist()
+
+    def test_export_no_pixels(self):
+        zip_archive = export_pixelsets(PixelSet.objects.none())
+        self._assert_archive_is_valid(zip_archive)
+
+        with zip_archive.open('meta.yaml') as meta_file:
+            meta_yaml = yaml.load(meta_file)
+
+            assert len(meta_yaml['pixelsets']) == 0
+
+        with zip_archive.open('pixels.csv') as pixels_file:
+            pixels_csv = pandas.read_csv(pixels_file).to_dict()
+
+            # Only the 'Omics Unit' column is present
+            assert len(pixels_csv) == 1
+
+    def test_export_pixels_without_analyses(self):
+        pixel_sets = factories.PixelSetFactory.create_batch(1)
+
+        zip_archive = export_pixelsets(pixel_sets)
+        self._assert_archive_is_valid(zip_archive)
+
+        with zip_archive.open('meta.yaml') as meta_file:
+            meta_yaml = yaml.load(meta_file)
+
+            assert len(meta_yaml['pixelsets']) == 1
+
+        with zip_archive.open('pixels.csv') as pixels_file:
+            pixels_csv = pandas.read_csv(pixels_file).to_dict()
+
+            # 'Omics Unit' column + 2 columns per pixel set
+            assert len(pixels_csv) == 2 * len(pixel_sets) + 1
```

Comment:
Add parenthesis to be more explicit (see remark below about operations precedence).